### DISCUSSION TITLE
[A/L] (2021/22) Logica - Esercizio 22

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#348] [Esercizi A-L]
![image](https://user-images.githubusercontent.com/69389461/152325557-afd0b170-f21c-49cb-9e35-0cf294d37040.png)

**Comment by Exyss** [comment]
![Esercizio 22](https://user-images.githubusercontent.com/65043207/152677782-1733d4fc-54a7-419a-933c-18e87a38fbdc.jpg)

**Comment by CiottoloMaggico** [comment]
![immagine](https://user-images.githubusercontent.com/79638284/210955289-e881772b-50dc-4765-87cd-81affbd86eb4.png)

**Comment by LuginiAndrea** [comment]
La domanda più semplice è:
Mi diresti di sì se la strada a destra è quella per la capitale?
La domanda è R, la domanda "interna" è I.
Vediamo perchè:

- La strada è quella destra e abbiamo un sincero: il sincero ci risponderebbe di sì a I, e quindi ci risponderebbe di sì ad R
- La strada è quella a sinistra e abbiamo un sincero: il sincero ci risponderebbe di no a I, e quindi no ad R
- La strada è quella a destra ed abbiamo un bugiardo: il bugiardo ci risponderebbe di no a I, e quindi ci risponderebbe di sì ad R
- La strada è quella a sinistra ed abbiamo un bugiardo: il bugiardo ci risponderebbe a I di si, e quindi ci risponderebbe di no ad R
$$I = p \times q \lor \neg p \times \neg q$$
Questo equivale al sì se è sincero e la strada e quella a destra oppure se è bugiardo ed è quella a sinistra.
Ora il ragionamento è semplice: se è sincero e dice sì, allora V. Se è bugiardo e dice no, allora è V.
Quindi:
$$R = p \times I \lor \neg p \times \neg I$$
